ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productName}.
+
+:standalone-server-type: ha
+:archiveType: war
+:uses-h2:
+:requires-multiple-servers:
+:jbds-not-supported:
+
+== What is it?
+
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productNameFull}.
+
+== Description
+
+The EJB remote calls contains a JTA transaction which is propagated over the EJB call.
+Further the quickstart demonstrates the transaction recovery which is run for both servers
+when a failure occurs.
+
+This quickstart contains two maven projects. As the goal is to make a remote call from one server
```

Comment:
"As the goal is to make a remote call"

I think you can delete this sentence, as you have describe earlier that the intent is to demonstrate calls between two servers.  

If you keep it, it is currently a sentence fragment.  Replace the period in the previous sentence with a comma, and make "as" lower-case.

Reply:
+1 for removal